content/millennium/r4/encounters/encounter.md

```diff
@@ -136,7 +136,7 @@ Create an individual Encounter.
 
 ### Headers
 
-<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Content-Type': 'application/fhir+json'} %>
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Accept': 'application/fhir+json', 'Content-Type': 'application/fhir+json'} %>
```

Comment:
Why is there an Accept header on a Create?

Reply:
We're relatively inconsistent in our exmaples for creates https://fhir.cerner.com/millennium/r4/individuals/patient/#create we either need to add this here or remove it from other places.

Reply:
Hmmmm. So best practice, I would think it would be to have an Accept header. The reason you dont need one is because we default to json. So if we have it in other places, then I think it's fine to leave it.